legacy/firmware/signing.c

```diff
@@ -72,6 +72,7 @@ static uint32_t idx1;      // The index of the input or output in the current tx
 static uint32_t idx2;  // The index of the input or output in the original tx
                        // (Phase 1), in the previous tx (Phase 2) or in the
                        // current tx when computing the legacy digest (Phase 2).
+static uint8_t external_inputs[64];  // bitfield of external input indices
```

Comment:
tiny hyperoptimization: wouldn't it be better to use `uint32_t` since we're running on a 32-bit processor?

(i don't know if the compiler is clever enough to optimize the bitmasking fun; it might be)

Reply:
https://github.com/trezor/trezor-firmware/pull/2144/commits/37bf82fdcb855801da6a1805068658e56ef484d8